src/templates/partials/navbar.hbs

```diff
@@ -64,7 +64,20 @@ component: navbar
           <a href="https://forum.bpmn.io" target="_blank" rel="noopener">Forum</a>
         </li>
         <li>
-          <a href="https://github.com/bpmn-io" target="_blank" rel="noopener">GitHub</a>
+          <a href="https://github.com/bpmn-io" target="_blank" rel="noopener" title="Explore our projects on GitHub">
+            <span class="visible-xs-inline" style="margin-right: .375em">GitHub</span>
```

Comment:
I'd rather use non-relative values for margin. Why does it need to depend on the font size?

Reply:
BTW it's a boomer we don't have CSP on the website, but it's a separate topic.

Reply:
> I'd rather use non-relative values for margin. Why does it need to depend on the font size?

It is the whitespace between text and label in mobile view (where font-size matters). Also, in bootstrap 5 and other UI libraries relative > absolute.

Reply:
Let it be as is then.